### PR TITLE
Removes js9

### PR DESCRIPTION
**Pull request by ashraffouda**
Signed-off-by: Ashraf Fouda <ashraf.m.fouda@gmail.com>

### CODE REVIEW

**Comment by ashraffouda** [comment]
https://github.com/0-complexity/openvcloud/issues/1680

File: 0-access.py [hunk 1]
```diff
@@ -117,19 +141,23 @@ def provision(remote):
...
-        for key in iyo_user_info["publicKeys"]:
-            j.tools.prefab.local.system.ssh.authorize(username, key["publickey"], **settings)
 
-    j.sal.fs.copyFile("/root/.ssh/id_rsa", "/home/%s/.ssh" % username)
-    j.sal.fs.chown("/home/%s/.ssh" % username, username, username)
-    j.sal.fs.writeFile("/home/%s/.remote" % username, "REMOTE=%s" % remote)
+        authorized_keys = "{ssh_dir}/authorized_keys".format(ssh_dir=ssh_dir)
+        for key in iyo_user_info["publicKeys"]:
+            authorize(username, key['publickey'], authorized_keys, **settings)
+    call(['cp', '/root/.ssh/id_rsa', "/home/%s/.ssh" % username])
```
Comment (grimpy) [inline 0-access.py]: shutil.copy

File: 0-access.py [hunk 2]
```diff
@@ -117,19 +141,23 @@ def provision(remote):
...
+    with open('/home/%s/.remote' % username, 'w+') as f:
+        f.write("REMOTE=%s" % remote)
+    call(['chown',  '-R', '{username}:{username}'.format(username=username), home])
```
Comment (grimpy) [inline 0-access.py]: os.chown

File: 0-access.py [hunk 3]
```diff
@@ -144,9 +172,8 @@ def provision(remote):
     database.add(ssh_session)
     database.commit()
 
-
     def _kill_session():
-        j.sal.fs.chown("/home/%s/.ssh" % username, "root", "root")
+        call(['chown', 'root:root', "/home/%s/.ssh" % username])
```
Comment (grimpy) [inline 0-access.py]: os.chown

File: flask_itsyouonline.py [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+def _get_info(username, access_token=None, jwt=None):
+    if access_token:
+        headers = {"Authorization": "token %s" % access_token}
+    else:
+        headers = {"Authorization": "Bearer %s" % jwt}
+    userinfourl = "https://itsyou.online/api/users/%s/info" % username
+    response = requests.get(userinfourl, headers=headers)
+    response.raise_for_status()
+    return response.json()
```
Comment (grimpy) [inline flask_itsyouonline.py]: why did we embed this file? it was part of the requirements no?

File: a.sh [hunk 5]
```diff
@@ -0,0 +1,4 @@
+source /root/.bashrc
+ZInstall_host_js9_full
```
Comment (grimpy) [inline a.sh]: what is this file doing here?

**Pull request merged**